<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputlineeditA">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputlineeditB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputlineeditC">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="answerlabel1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QLabel" name="answerlabel2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QLabel" name="answerlabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate the roots of a Polynomial</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerlabel1_4">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Value of A=</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerlabel1_5">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Value of B=</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerlabel1_6">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>150</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Value of C=</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerlabel1_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QLabel" name="answerlabel2_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QLabel" name="answerlabel1_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Coefficients</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Calculate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>calculatebuttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculatebuttonClicked()</slot>
 </slots>
</ui>
